client/src/shared/ui/Section.less

```diff
@@ -6,20 +6,20 @@
   max-height: var(--section-max-height);
   flex: inherit;
 
-  .section__header,
   .section__body {
-    padding: 12px;
+    padding: 12px 12px 0 12px;
+    margin-bottom: 12px;
```

Comment:
Why? What do we want to accomplish with this one?

Reply:
The padding is applied at the end of that div. Which means when it's not fully scrolled, there isn't any whitespace. See left (padding) and right (margin):
<div>
<img width="300px" src="https://user-images.githubusercontent.com/25825387/143211301-52fd60b2-058f-47c1-9d16-73d05e2807aa.png"/>

<img width="300px" src="https://user-images.githubusercontent.com/25825387/143211265-310e8f86-bca6-4ff1-aded-14eac2584b21.png"/>
 </div>

Reply:
My expectation as a user is to not have that padding, but have the scroll bar to the end of the section container. 

It complicates our styles and it does not match the default UX is see in apps and browsers around. Please clarify with @andreasgeier and re-consider potentially.

Reply:
Section at the moment is built in a way that it works as expected in browsers and apps (no bottom padding).

Reply:
The basic rule here is that the layout should not change on overflow. When we have whitespace between a section start/end and its content, we keep that space in the overflow scenario. Another example is the file list for deployment. So, let's ensure we have this expected behavior in place.

https://user-images.githubusercontent.com/938244/143227596-48452007-c2df-4443-a908-7095e591dbd4.mov

Reply:
What about the following screen: 

![capture ehjNzh_optimized](https://user-images.githubusercontent.com/58601/143236763-e0560989-cba7-4f19-9c38-940bf1f84fc7.gif)

You clearly see that we do have the expected behavior (from my point of view) in place in  place everywhere.

This is how I'd imagine it to work for sections, too: 

![capture IQsYnK_optimized](https://user-images.githubusercontent.com/58601/143237062-985d9951-2e91-4869-9057-a24a41510f2d.gif)


Note that it also does not break the additional files list you're mentioning:

![capture 3Zt4Wo_optimized](https://user-images.githubusercontent.com/58601/143237492-3a340e09-6040-4387-b536-e2c64ba64a04.gif)

On MacOS this is less visible; on any other platform having a padding below a scrollable container is visual disturbing.

Reply:
@nikku Your examples are different scenarios. You can't compare a table scrolling with a content section. I believe the file list example makes it very clear, I don't want to have this list docked on the deploy button. 
I hear and appreciate your feedback. But, let's conclude this discussion and keep it as designed.

Reply:
Thanks for clarifying :+1: